### PR TITLE
inital implementation of the crash scenario

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Content "Added new starting scenario, the Road Accident"

#### Purpose of change

A new scenario, to add flavour, distinct from current scenarios.

#### Describe the solution

Added new scenario: Road accident

- Premise: With the cataclysm coming, there would be quite a few people making a break for it via car. The player in this scenario is one of these people.

With their car forced off the road, the player crashes, and wakes up in the ruined vehicle.

#### Describe alternatives you've considered

N/A

#### Testing

Start game. Check scenario works. Load scenario.

#### Additional context

- [x] import a "wreckage" vehicle
- [x] create map extra
- [x] add item groups for crash site
- [x] add monsters

Mock-up: 
image

Also,[ this discussion on Reddit,](https://www.reddit.com/r/cataclysmdda/comments/83sroh/new_scenario_idea_road_accident/) way back in 2019, shows the full extent of the scenario i had wanted to recreate.

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
If it's a work in progress, consider converting the PR into a draft.

File: data/json/scenarios.json [hunk 1]
```diff
@@ -574,5 +574,16 @@
     "allowed_locs": [ "sloc_private_resort" ],
     "professions": [ "frat", "gangster", "lost_sub", "lawyer", "politician", "dancer" ],
     "flags": [ "CHALLENGE", "LONE_START" ]
+  },
+    {
+    "type": "scenario",
+    "id": "road_accident",
+    "name": "Road Accident",
+    "points": -3,
+    "description": "You decided that the safety of the open road would be a better prospect than the zombie-filled cities.  Driving out of town, you swerved off the road and crashed.  The situation is looking grim",
```
Comment (Night-Pryanik) [inline data/json/scenarios.json:583]:
```suggestion
    "description": "You decided that the safety of the open road would be a better prospect than the zombie-filled cities.  Driving out of town, you swerved off the road and crashed.  The situation is looking grim.",
```

File: data/json/mapgen/map_extras/car_crash.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_car_crash",
+    "object": {
+      "place_vehicles": [ { "vehicle": "road_straight_wrecks", "x": 17, "y": 6, "chance": 100, "rotation": 270 } ],
```
Comment (Maleclypse) [inline data/json/mapgen/map_extras/car_crash.json:7]:
```suggestion
      "place_vehicles": [ { "vehicle": "wreckage_scenario", "x": 17, "y": 6, "chance": 100, "rotation": 270 } ],
```

File: data/json/mapgen/map_extras/car_crash.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 0, 23 ], "y": [ 0, 23 ], "repeat": [ 1, 5 ], "density": 0.1 } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_car_crash",
+    "object": {
+      "place_vehicles": [ { "vehicle": "road_straight_wrecks", "x": 12, "y": 12, "chance": 100, "rotation": 270 } ],
```
Comment (Maleclypse) [inline data/json/mapgen/map_extras/car_crash.json:16]:
```suggestion
      "place_vehicles": [ { "vehicle": "wreckage_scenario", "x": 12, "y": 12, "chance": 100, "rotation": 270 } ],
```

**Review by Maleclypse — COMMENTED** [review]
The test was saying "road_straight_wrecks" isn't a valid vehicle.  So I made an adjustment to place the wreckage vehicle that you created.  what's the "type" for "road_straight_wrecks"?

**Comment by Xenomorph-III** [comment]
road_straight_wrecks is a vehicle group that is used for spawning the wreckages in cities.

**Comment by Maleclypse** [comment]
Should this PR be closed as there is a merged car crash scenario?

**Pull request closed without merging**